crates/burn-core/src/nn/transformer/pwff.rs

```diff
@@ -36,13 +36,32 @@ pub struct PositionWiseFeedForwardConfig {
 ///
 /// Should be created using [PositionWiseFeedForwardConfig]
 #[derive(Module, Debug)]
+#[module(custom_display)]
 pub struct PositionWiseFeedForward<B: Backend> {
     linear_inner: Linear<B>,
     linear_outer: Linear<B>,
     dropout: Dropout,
     gelu: Gelu,
```

Comment:
We should put the parameters public.

Reply:
Fixed. I made other modules' attributes public for consistency since it's been requested frequently.